x-pack/plugins/cases/public/components/all_cases/use_cases_columns.tsx

```diff
@@ -49,23 +48,6 @@ type CasesColumns =
   | EuiTableComputedColumnType<CaseUI>
   | EuiTableFieldDataColumnType<CaseUI>;
 
-const LINE_CLAMP = 3;
-const LineClampedEuiBadgeGroup = euiStyled(EuiBadgeGroup)`
```

Comment:
Removing this logic broke the `tags` in the cases table.

There is another place where we used the `LINE_CLAMP` logic but I don't remember where that one is used 😅 

In this case, we had this deliberately here to prevent rows in the cases table from growing too much.

<details><summary>Before</summary>
<img width="667" alt="Screenshot 2024-03-14 at 15 32 55" src="https://github.com/elastic/kibana/assets/1533137/3db28150-d0f7-4b05-ba6e-a80c55b21cee">
</details>

<details><summary>Now</summary>
<img width="667" alt="Screenshot 2024-03-14 at 15 32 43" src="https://github.com/elastic/kibana/assets/1533137/71e252d3-ae97-4e42-a6a1-cf8da1fbc005">
</details>

So two things:
1. The lines grow vertically indefinitely when we add more tags.
2. Tags don't seem to have ellipses inside them anymore.

Reply:
Follow up: Looking a bit below it seems this is because you do `<TruncatedText text={tag} />` on line 140. This allows for the tags themselves to be too long preventing the ellipses.

Additionally, this previous logic "clamped" the collection of tags and not individual tags so that's prob why now we can have >3 lines.

Reply:
Fixed in https://github.com/elastic/kibana/pull/178379/commits/0f2559c2d21683d607100248d4d89c955c3123dd